### ISSUE TITLE
Support for Cool-only systems

### ISSUE THREAD

**Issue by jlrobbin** [fjs21/homebridge-kumo#68]
Hi, my system doesn't have heating, just cooling. And that's setup correctly in Kumo cloud. Wondering if there is a way to disable the heating option or have it respect the information coming from Kumo?

**Comment by fjs21** [comment]
In your Homebridge log, do you see anything like:

> 'Initializing "%s" of unitType "%s" as generic (unspecified) unit.'

Where the first %s is the name of your device.

If so, what is stated as the unitType? I use that to identify the unit - so if your's is new then I can try to develop a new accessory that only supports cooling.

**Comment by jlrobbin** [comment]
I see this: "Initializing "Main Bedroom" as ductless unit.". In my Kumo app, there are settings in advanced for each unit to specify system type.  "Cool only" or "Cool and Heat" are the two choices.

**Comment by jasongin** [comment]
I wanted this capability also, so I implemented support for disabling certain functions via plugin options at https://github.com/jasongin/homebridge-kumo/commit/89177911c97cbd97798a3d42537e18fda90f9dcc

If you use that to disable heating, the effect is that you no longer see a heat/cool/auto mode selector in the UI, just a power button that turns on/off the cooling. I'll bring this in as a pull request very soon; I'm waiting on another PR first.
